### PR TITLE
update critical tasks in taskotron rules

### PR DESCRIPTION
**Pull request by kparal**
Hello, please also include this change in the upcoming FMN release.

### CODE REVIEW

File: fmn/rules/taskotron.py [hunk 1]
```diff
@@ -5,7 +5,8 @@
 RELEASE_CRITICAL_TASKS = [
```
Comment (pypingou) [inline fmn/rules/taskotron.py:5]: Suddenly I wonder if it would make sense to have these in a configuration file :)

**Comment by pypingou** [comment]
One comment, otherwise looks good to me

File: fmn/rules/taskotron.py (hunk 1 above)
Comment (kparal) [reply fmn/rules/taskotron.py:5]: That's up to the fmn devs, I guess. If they prepare some plumbing, we can certainly adjust the code.

However, I'd like to push this change now, and deal with config files later.

Comment (jeremycline) [reply fmn/rules/taskotron.py:5]: It does make sense to me for this to be in a configuration file, but currently the configuration just using fedmsg's configuration and needs a lot of refactoring anyway. I'll file an issue about it.

**Review by jeremycline — APPROVED** [review]
I filed #199 for the configuration issue, but otherwise this looks fine. Thanks!

**Comment by kparal** [comment]
Thanks! That's all we needed wrt taskotron changes before a new release.

**Comment by kparal** [comment]
I looked at staging and the updated rule documentation doesn't render properly. I wonder why?

fmn

**Comment by pypingou** [comment]
Could it be because of the ``::`` which normally indicate a code-block?

**Comment by kparal** [comment]
Thanks @pypingou . Already resolved in #203.

**Pull request merged**